src/utils/text_display.jl

```diff
@@ -27,6 +28,11 @@ end
 function human_readable_name(name::Vector{Symbol}; title_case::Bool=false)::Vector{String}
     return human_readable_name(String.(name); title_case=title_case)
 end
+function human_readable_name(
+    names::NTuple{N,Symbol}; title_case::Bool=false
+)::NTuple{N,String} where N
+    return human_readable_name.(String.(names); title_case=title_case)
+end
```

Comment:
I think I prefer either mandating that the `names` argument should always be a `Vector{Symbol}` or expanding the allowable data type on the existing implementation.

Currently you have duplicate implementations

```julia
function human_readable_name(names::Union{Vector{T}, NTuple{N, T}}; title_case::Bool=false)::Vector{String} where {T<:Symbol,N<:Int64}
    return human_readable_name(String.(name); title_case=title_case)
end
```

PS: I could be wrong but I think `::NTuple{N, String} where N` may be equivalent to `NTuple{Any, String}`

Reply:
about the PS:
![image](https://github.com/user-attachments/assets/263a6097-57c4-4575-88aa-5d2735e00ce6)

Reply:
What I meant was that `N` is interpreted to be `Any`. I wasn't claiming that it was valid syntax

Reply:
Oh, I see. So, is there a way (that you know of) of declaring a NTuple without specifying the number of elements?

Reply:
In my snippet above, I declare `N` as only subtyping `Int64` so the compiler can specialise on that

Reply:
1. I just updated this code to avoid the duplicate implementation that Takuya was mentioning
2. With the `where {T<:Symbol, N<:Integer}` if you have multiple inputs you can assure that they have the same element type. Also, `T` and `N` are available to you inside the scope of your function in case you need to variables with the same type/element type as T or N.